cpp/src/Slice/Parser.cpp

```diff
@@ -2151,16 +2151,16 @@ Slice::Container::visitContents(ParserVisitor* visitor)
 }
 
 void
-Slice::Container::checkIntroduced(const string& scopedName, ContainedPtr namedThing)
+Slice::Container::checkIntroduced(const string& name, ContainedPtr namedThing)
```

Comment:
Unrelated cleanup from my last PR.

It feels weird to have a parameter named `scopedName`, when the first thing the function does is check if it's actually unscoped...